src/components/paragraph/sul-home-banner/sul-home-banner.client.tsx

```diff
@@ -27,9 +28,37 @@ export const SulHomeBannerRandomClient = ({children, ...props}: Props) => {
 export const SulHomeBannerFormClient = () => {
   const [formAction, setFormAction] = useState("/all")
   const inputId = useId()
+  const formRef = useRef<HTMLFormElement>(null)
+
+  // Google Analytics: Track the selected search option when the user submits the form.
+  // Helps measure actual search behavior by logging which resource option was used.
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault()
+
+    // @TODO: DELETE BEFORE MERGE
+    console.log("Form submission prevented. Firing GA event:", formAction)
+
+    sendGAEvent({
+      event: "search_option_selected",
+      category: "Search",
+      label: formAction,
+    })
+
+    // Delay to let GA event send before navigation
+    setTimeout(() => {
+      // @TODO: DELETE BEFORE MERGE
+      console.log("Now submitting the form.")
+
+      formRef.current?.submit()
+    }, 300)
+  }
+
   return (
     <form
+      ref={formRef}
       action={formAction}
+      onSubmit={handleSubmit}
+      method="GET"
```

Comment:
This isn't _really_ necessary since forms default to `GET`. But I assume it's needed for the `formRef.current?.submit()`?

Reply:
Great question! I did a little more reading and it's not required, just recommended 😄 I went ahead and removed it!